sim/battle.ts

```diff
@@ -2614,11 +2614,16 @@ export class Battle {
 		);
 
 		for (let i = 0; i < this.sides.length; i++) {
-			if (switches[i] && !this.canSwitch(this.sides[i])) {
+			let reviveSwitch = false; // Used to ignore the fake switch for Revival Blessing
+			if (switches[i] && !this.canSwitch(this.sides[i]) && !reviveSwitch) {
```

Comment:
`&& !reviveSwitch` is redudant because you define that as `false` on the previous line.

Reply:
Yeah that's valid